pkg/gce-cloud-provider/compute/gce-compute.go

```diff
@@ -90,18 +90,33 @@ func (cloud *CloudProvider) GetDefaultZone() string {
 // ListDisks lists disks based on maxEntries and pageToken only in the project
 // and zone that the driver is running in.
 func (cloud *CloudProvider) ListDisks(ctx context.Context, maxEntries int64, pageToken string) ([]*computev1.Disk, string, error) {
-	lCall := cloud.service.Disks.List(cloud.project, cloud.zone)
```

Comment:
I think we want to drop the maxEntries & pageToken parameters if we're just going to drop them. 

It also looks like we may need to handle paging a step up in ListVolumes, as the caller of that is going to only expect maxEntries results.

Reply:
agreed that we might just drop maxEntries and pageToken
we can handle the paging in ListVolume by maintaining a list in memory and refresh it whenever there's no pageToken passed in, but I think we wouldn't need to deal with that as long as [attacher](https://github.com/kubernetes-csi/external-attacher/blob/0691388e9b94c43ac0aea6840fa81662b5741722/pkg/attacher/lister.go#L39) remains to be the only caller of this function.

Reply:
Technically some other container orchestrator may be using this driver so we don't want to break the API.

Reply:
done